memory.js

```diff
@@ -1,14 +1,23 @@
 /* eslint-disable camelcase */
+var MessageChannel = global.MessageChannel
+if (MessageChannel == null) ({ MessageChannel } = require('worker' + '_threads'))
```

Comment:
@emilbayes Is this a hack to please browserify? When I use `-p tinyify` with browserify this crashes for some unknown reason.

Reply:
If I change those strings to `'worker_threads'` then also browserify crashes, without using `tinyify`, so it doesn't seem related to `tinyify`.